.craft.yml

```diff
@@ -21,13 +21,13 @@ targets:
     gradleCliPath: ./gradlew
     mavenCliPath: scripts/mvnw
     mavenSettingsPath: scripts/settings.xml
-    mavenRepoId: ossrh
-    mavenRepoUrl: https://oss.sonatype.org/service/local/staging/deploy/maven2/
+    mavenRepoId: ossrh-staging-api
+    mavenRepoUrl: https://ossrh-staging-api.central.sonatype.com/service/local/staging/deploy/maven2/
     android: false
     includeNames: /sentry-kotlin-multiplatform-gradle-plugin.*$/
   - name: github
   - name: registry
     sdks:
       maven:io.sentry:sentry-kotlin-multiplatform:
-#      maven:io.sentry:sentry-kotlin-multiplatform-gradle-plugin:
+      maven:io.sentry:sentry-kotlin-multiplatform-gradle-plugin:
```

Comment:
we forgot to uncomment it after the first release right? 🙈

Reply:
oopsie, I dont think it mattered much though since its synced with the sdk versions

but yeah we should uncomment it